cmd/ctlog/createctconfig/main.go

```diff
@@ -96,7 +96,7 @@ func main() {
 	client := fulcioclient.NewClient(u)
 	root, err := client.RootCert()
 	if err != nil {
-		logging.FromContext(ctx).Panicf("Failed to fetch fulcio Root cert")
+		logging.FromContext(ctx).Panicf("Failed to fetch fulcio Root cert: %v", err)
```

Comment:
i think you can use `%w` instead of `%v`
also please sign the DCO

Reply:
Done